dashboard/app/controllers/musiclab_controller.rb

```diff
@@ -1,5 +1,14 @@
 class MusiclabController < ApplicationController
   ANALYTICS_KEY = CDO.amplitude_api_key
+  NUM_MINI_PLAYER_PROJECTS = 5
+  # Hard-coded list of channel IDs for the mini player if DCDO flag is set to `true`
+  CHANNELS = %w(
+    syBuoFelbGB3eOmNVoQGrWMXEk0l1EhkIX6c08ujq6s
+    Ehnks69B0Whcn_YQQNCK4GUHAPU3WSG2jfilvQF1kfo
+    6Xc53NIhwFxjjwsSaoj_eSiRbbXr97BYQ3W_7vIaAwY
+    NwTkJSskTswEOtgy6TbaJ-8SonhhSxojrJjlJLBko4w
+    I3HQObQN0jiadlMqjleet5VEwQaFwGYHG6Ehxodbg7U
```

Comment:
FYI @bencodeorg - I updated this last channel id because there was not a user id associated with it. So I remixed project and added that new channel id so it could be published/featured.